docs/user/guides/troubleshooting/index.rst

```diff
@@ -1,14 +1,22 @@
-Guides for troubleshooting problems
------------------------------------
+How-to guides: troubleshooting problems
+---------------------------------------
 
 In the following guides,
 you can learn how to fix common problems using Read the Docs.
 
+⏩️ :doc:`../build-troubleshooting`
```

Comment:
```suggestion
⏩️ :doc:`/guides/build-troubleshooting`
```
This is so much cleaner 😍 

I like using full paths, it will break much less often.

Reply:
I was in the business of copy-pasting, thanks for fixing :)

The old layout used relative paths, so I didn't get around to fixing all of them.